src/officehours/settings.py

```diff
@@ -78,6 +78,11 @@ def str_to_bool(val):
     ]
 
 WATCHMAN_TOKENS = os.getenv('WATCHMAN_TOKENS')
+WATCHMAN_TOKEN_NAME = os.getenv('WATCHMAN_TOKEN_NAME', 'officehours-watchman-token')
+WATCHMAN_CHECKS = ('watchman.checks.caches', 'watchman.checks.databases')
```

Comment:
I feel based on the Matt comment these should be configurable, I think add these 2 variables to `.env.sample`, uncomment here and make it default to `False`. So if anybody did not add it to `.env` the defaults are applied.

Something like this should work, 

config_to_bool = lambda value: str(value).lower() in ('true', '1', 'yes', 'on')
WATCHMAN_DISABLE_APM = config_to_bool(os.getenv('WATCHMAN_DISABLE_APM', False))

Reply:
alright, I think its ok now? thank you for the feedback.